### ISSUE TITLE
[idea] Porting to WebAssembly

### ISSUE THREAD

**Issue by YordanYanakiev** [Mapsui/Mapsui#714]
Hello.
I am just excited with Blazor and WebAssembly, and just as an idea - it could be cool to have it the MapsUI as Web port written in C# for Blazor and/or eventually as WebAssembly.
( Skyasharp is having port now to Blazor )
Seems like Microsoft have some great plans for Blazor as complete crossplatform framework in C# (Web/Desktop/Mobiles) so.. maybe it would be cool to have the awesome maps in there as well. :)

**Comment by pauldendulk** [comment]
Yes, that is an idea that was in the back of my mind. I also notices SkiaSharp was used in Blazor. I have no idea how that performs. This is crucial factor for Mapsui in Blazor.

Blazor already supports .net standard 2.0. Most of Mapsui is in .net standard 2.0 so that part should be easy.
https://docs.microsoft.com/nl-nl/aspnet/core/blazor/?view=aspnetcore-3.0#code-sharing-and-net-standard

Also I don't know how Blazor will develop further. Here a google trends compare between Blazor and Xamarin. 

https://trends.google.com/trends/explore?date=today%205-y&q=blazor,Xamarin

**Comment by YordanYanakiev** [comment]
Wow! Happy to see that You are already on it! :)
Well... according to Microsoft ( at least their Blazor team ) - they maybe.. eventually going to have 2 cross-platforms development frameworks - Blazor and Xamarin.
https://visualstudiomagazine.com/articles/2019/09/26/blazor-future.aspx
( currently seems like they are testing Blazor + Flutter for mobile development )
anyway separate project or/and sharing the same codebase would be awesome.

**Comment by pauldendulk** [comment]
Well, I have only been thinking about it. Lots of other things to do first. Let's see where Blazor is going.
